tests/automation_framework/data/work_order/work_order_submit_encryptedDataEncryptionKey_not_set_echoClient.json

```diff
@@ -0,0 +1,35 @@
+{
+    "jsonrpc": "2.0",
+    "method": "WorkOrderSubmit",
+    "id": 11,
+    "params": {
+        "responseTimeoutMSecs": 6000,
+        "payloadFormat": "json-rpc",
+        "resultUri": "resulturi",
+        "notifyUri": "notifyuri",
+        "workOrderId": "",
+        "workerId":"",
+        "workloadId": "echo-result",
```

Comment:
workload_Id is echo_result but the Data parameters are for heart disease eval.

Reply:
same as above.